### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by ForHemer**
In this project, I will build a simple HTML list of To Do tasks. This simple web page will be built using webpack and served by a webpack dev server
Screenshot
.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
...
+#### View pages from the browser
+
+- If you just want to check the webpage/webapp, you have it available on the live demo link.
+
+#### Install locally the code of the application
+
+In order to install a local version of this project and please do the following steps:
+- Install the latest version of any code editor.
+- install the latest version of GIT.
+- Go to the [repository](https://github.com/ForHemer/To-Do-List) and clone it.
```
Comment (Gambit142) [inline README.md]: - [x] It would be nice to include information about how to clone, install and run the server locally. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @ForHemer 👋, 

### Highlights
- Proper implementation of the list structure ✔️ 
- Clean and organized code ✔️ 

Good job so far!

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Whoistolu) [reply README.md]: I can see that this has been resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @ForHemer,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**